javascript/javascript1/week3/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>week3</title>
+    <script src="index.js"></script>
```

Comment:
please read why and where should keep js either in the head or at end of the body.

Reply:
i have modified it locally but if i want to commit and push these changes do i need to make the branch js/js/week3 again ? or i just need to add commit and push after modifying ?